apps/dotcom/sync-worker/src/utils/tla/getAuth.ts

```diff
@@ -3,15 +3,35 @@ import { IRequest, StatusError } from 'itty-router'
 import { Environment } from '../../types'
 
 export async function requireAuth(request: IRequest, env: Environment): Promise<SignedInAuth> {
-	const auth = await getAuth(request, env)
+	const auth = await getAuthFromSearchParams(request, env)
 	if (!auth) {
 		throw new StatusError(401, 'Unauthorized')
 	}
 
 	return auth
 }
 
-export async function getAuth(request: IRequest, env: Environment): Promise<SignedInAuth | null> {
+export async function getAuthFromCookies(
+	request: IRequest,
+	env: Environment
+): Promise<SignedInAuth | null> {
+	const clerk = createClerkClient({
+		secretKey: env.CLERK_SECRET_KEY,
+		publishableKey: env.CLERK_PUBLISHABLE_KEY,
+	})
+
+	const state = await clerk.authenticateRequest(request)
+	if (!state.isSignedIn) {
+		return null
+	}
+
+	return state.toAuth()
+}
```

Comment:
did the existing `getAuth` not work with cookies? seems like it should have done. no biggie either way. plus I guess it's easier to read when we explicitly ask for a particular method of token passing.